dmriprep/interfaces/vectors.py

```diff
@@ -97,3 +98,62 @@ def _undefined(objekt, name, default=None):
     if not isdefined(value):
         return default
     return value
+
+
+class _ReorientVectorsInputSpec(BaseInterfaceInputSpec):
+    in_rasb = File(exists=True)
+    affines = traits.Array()
+    b0_threshold = traits.Float(B0_THRESHOLD, usedefault=True)
+
+
+class _ReorientVectorsOutputSpec(TraitedSpec):
+    out_rasb = File(exists=True)
+
+
+class ReorientVectors(SimpleInterface):
+    """
+    Reorient Vectors
+
+    Example
+    -------
+
+    >>> os.chdir(tmpdir)
+    >>> oldrasb = str(data_dir / 'dwi.tsv')
+    >>> oldrasb_mat = np.loadtxt(str(data_dir / 'dwi.tsv'), skiprows=1)
+    >>> # The simple case: all affines are identity
+    >>> affine_list = np.zeros((len(oldrasb_mat[:, 3][oldrasb_mat[:, 3] != 0]), 4, 4))
+    >>> for i in range(4):
+    ...     affine_list[:, i, i] = 1
+    ...     reor_vecs = ReorientVectors()
+    >>> reor_vecs = ReorientVectors()
+    >>> reor_vecs.inputs.affines = affine_list
+    >>> reor_vecs.inputs.in_rasb = oldrasb
+    >>> res = reor_vecs.run()
+    >>> out_rasb = res.outputs.out_rasb
+    >>> out_rasb_mat = np.loadtxt(out_rasb, skiprows=1)
+    >>> npt.assert_equal(oldrasb_mat, out_rasb_mat)
```

Comment:
We need to import `npt` before reaching this line  (see: https://travis-ci.org/nipreps/dmriprep/jobs/637522010#L577)

Reply:
On further thought, maybe replace this with something like: 

```
>>> oldrasb_mat == out_rasb_mat
 True
```